offsite_payments.gemspec

```diff
@@ -22,9 +22,20 @@ Gem::Specification.new do |s|
   s.files = Dir['CHANGELOG', 'README.md', 'MIT-LICENSE', 'lib/**/*']
   s.require_path = 'lib'
 
+  unless defined?(Money)
+    s.post_install_message = %q{
+    In order to use `offsite_payments` gem, you need to either install or add to your Gemfile
+    one of the two options for Money gem:
+      - `gem 'money'`
+      - `gem 'shopify-money', require: 'money'`
+
+    Regardless of which gem you choose, please add them *before* `offsite_payments`.
+    For more info, go to https://github.com/activemerchant/offsite_payments#money-gem-dependency
+    }
```

Comment:
This should include a message about any Money gem usage being considered deprecated, encouraging people to migrate to use their own Money gem e.g.` Money.from_amount(notification.gross)` instead of this gem doing it for them.

I'd like to resume work on https://github.com/activemerchant/offsite_payments/pull/248 since we want to move away from this gem mess 🙊

Reply:
👍